WTraining/app/controllers/user.js

```diff
@@ -23,3 +22,24 @@ exports.createUser = (req, res, next) => {
       return next(e);
     });
 };
+
+exports.signIn = (req, res, next) =>
+  userService
+    .findOneUser(req.body.email)
+    .then(user => {
+      if (!user) throw unauthorizedError('wrong user or password');
```

Comment:
I think that it is a good idea to use an interactor because in the signIn controller there are a lot of things. What do you think ?